CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
```

Comment:
Please add a link to the proposal document at the bottom of the ChangeLog, so this will become a real link

Reply:
Ah, thank you!

Reply:
Added.